everest-testing/src/everest/testing/ocpp_utils/fixtures.py

```diff
@@ -161,10 +123,10 @@ async def charge_point_v16(request, central_system_v16: CentralSystem, test_cont
 
 
 @pytest_asyncio.fixture
-async def charge_point_v201(central_system_v201: CentralSystem, test_controller: TestController):
+async def charge_point_v201(central_system_v201: CentralSystem, test_controller: EverestTestController):
     """Fixture for ChargePoint16. Requires central_system_v201 and test_controller. Starts test_controller immediately
     """
-    test_controller.start(central_system_v201.port)
+    test_controller.start()
     cp = await central_system_v201.wait_for_chargepoint()
     yield cp
     cp.stop()
```

Comment:
note: I keep getting warnings that this is not getting awaited. Should we maybe also fix that?